Added summary section at the end (#926)
Agoric/documentation: main/guides/getting-started/README.md

![Making an offer](./assets/new_004.png)

When the transaction is complete you will notice some IST has been debited from your wallet, and you are the owner of three new properties. Feel free to experiment with offering less than the required 0.25 IST or attemping to purchase more than three items. You'll see these offers will not be accepted. Conditional offers like this are a key part of the Agoric platform!

![Finished transaction](./assets/new_006.png)

Congratulations! You got your first Agoric dapp up and running! Now that you've got your first Agoric dapp up and running, let's focus on some key takeaways from this exercise:

> - Starting an Agoric Chain: You have learned the process to initialize and start a local Agoric blockchain. 
> 
> - Deploying a Contract: You have learned the steps to deploy a smart contract onto the Agoric platform!

Reviewer: yes, input from @sufyaankhan seems in order.

meanwhile, one other idea:

```suggestion
- **Starting an Agoric Chain**: You have gained experience starting a local Agoric blockchain. 

- **Deploying a Contract**: You have deployed a smart contract onto the Agoric platform!
```

> - Making an Offer: Finally, you've learned to make an offer and seen the conditional nature of the offers that can be made on the Agoric platform. As you saw, when attempting to submit less than 0.25 IST or purchase more than three items the offer is rejected.

Reviewer: I like the "Feel free to experiment ..." bit above, but the key take-away is not that other offers are rejected. The contract itself could be doing that. The key takeaway is that **the constraints they sign are enforced by the platform**.

The ui can't spoof them by showing a 25 cent charge but really charging 500 IST - that would show up in the Keplr dialog. Nor can the contract take the 0.25 cents without satisfying the want constraint. And neither can the contract take more than the 0.25 cents in the give contraint.

Reviewer: Leaving out the "As you saw..." sentence would address my concern for this PR.

Author: Points about the "Making an Offer" bullet in Summary section addressed

> - **Making an Offer**: Finally, you've learned to make an offer and seen that offer constraints are enforced by the Agoric platform. This means user interface can't attempt to fool a user by showing a 0.25 IST when actually charging them a much higer amount, such as 500 IST. It's also important to note that the contract cannot take the 0.25 IST without satisfying the want constraint, nor can the contract take more than the 0.25 cents in the give contraint.

Reviewer: Oh... we really do need that Keplr offer screenshot to go with this point.

 - https://github.com/Agoric/documentation/issues/921

![Finished transaction](./assets/new_006.png)

Congratulations! You got your first Agoric dapp up and running! Now that you've got your first Agoric dapp up and running, let's focus on some key takeaways from this exercise:
- Starting an Agoric Chain: You have learned the process to initialize and start a local Agoric blockchain. 
```sh
yarn start:docker

> ```

Reviewer: hm. I wouldn't call that **the** process to start a local chain. `yarn start:docker` is not a general purpose mechanism at all, and we didn't really teach them how it works.

Maybe...

----

Congratulations! You got your first Agoric dapp up and running!

Key takeaways:

 - start a local Agoric blockchain
 - deploy a smart contract
 - make an offer, using a web ui and Keplr extension
----

Hm... I don't love that either.

Author: I think we need @sufyaankhan's input on this one. The code examples I added in this section were the lines from this doc - that's how I understood @sufyaankhan's request in Item 5 of #919

- Deploying a Contract: You have learned the steps to deploy a smart contract onto the Agoric platform!
```sh

> yarn start:contract

Reviewer: Again, these are not **the** steps to deploy a contract. There are lots of other ways.

I'm still interested in feedback on #906 . If we incorporated that, we'd be in a better position to say that readers learned something about deployment. With what we have now, they do manage to get it done, but I wouldn't say that they learn how it works.

Author: Just approved #906, I think it resolves the need for the code one-liners here. I didn't see that one in my notifications though - is that because it was in a draft state? Any way to get draft state notifications?

Reviewer: I think I get the same notifications for draft as non-draft, but who knows whether that's cuz I tweaked something.

```

> - Making an Offer: Finally, you have learned how to make offers within the dapp and add items to your Keplr wallet!

Reviewer: I'd rather not say that adding Keplr accounts is a key take-away from our docs. That's something Keplr documents pretty well.

And again, I'm more comfortable saying that they managed to make an offer than that they learned anything about it.

In the video, I'd like us to focus a bit more on the offer step: try to make an offer with <0.25 IST and see what the contract says. Likewise, make an offer with >3 items. And make the point that what the user is signing is a _conditional_ offer. Regardless of how the dapp and the contract behave, the give / want constraints that they sign in Keplr are enforced by the platform.

We could put a bit more of that in the prose. With that, we could also add here that they learned about the conditional nature of offers on the Agoric platform.

Author: @dckc:
- Re-worded that part
- Removed code snippets
- Added content in "Making an Offer" section about bidding <0.25 IST or purchasing >3 items
- Will include this in new video / screenshots as well
